github/github.go

```diff
@@ -501,6 +501,15 @@ func (r *RateLimitError) Error() string {
 		r.Response.StatusCode, r.Message, r.Rate.Reset.Time.Sub(time.Now()))
 }
 
+// AcceptedError occurs when GitHub returns 202 Accepted response with an
+// empty body, which means a job was scheduled on the GitHub side to process
+// the information needed and cache it.
+type AcceptedError struct{}
+
+func (r *AcceptedError) Error() string {
```

Comment:
Receiver is unused and can be removed. See rationale [here](https://dmitri.shuralyov.com/idiomatic-go#avoid-unused-method-receiver-names).